src/test/java/io/phasetwo/service/resource/OrganizationResourceTest.java

```diff
@@ -451,6 +451,100 @@ void testAddGetDeleteMemberships() throws IOException {
+
+    // search members with query parameter
+    response = getRequest(id, "members?search=john");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(2));
+    assertThat(members, hasItem(hasProperty("username", is("johndoe"))));
+    assertThat(members, hasItem(hasProperty("username", is("johndow"))));
+
+    response = getRequest(id, "members?search=jack,jill");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(2));
+    assertThat(members, hasItem(hasProperty("username", is("jack"))));
+    assertThat(members, hasItem(hasProperty("username", is("jill"))));
+
+    response = getRequest(id, "members?search=john, jack, jill");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(4));
+    assertThat(members, hasItem(hasProperty("username", is("johndoe"))));
+    assertThat(members, hasItem(hasProperty("username", is("johndow"))));
+    assertThat(members, hasItem(hasProperty("username", is("jack"))));
+    assertThat(members, hasItem(hasProperty("username", is("jill"))));
+
+    response = getRequest(id, "members?search=,,jack");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(1));
+    assertThat(members, hasItem(hasProperty("username", is("jack"))));
+
+    response = getRequest(id, "members?search=,, ,");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(0));
+
+    response = getRequest(id, "members?search= ,, jack , ");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(1));
+    assertThat(members, hasItem(hasProperty("username", is("jack"))));
+
+    response = getRequest(id, "members?search=");
+    assertThat(response.statusCode(), is(Status.OK.getStatusCode()));
+    members = objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(members, notNullValue());
+    assertThat(members, hasSize(5));
```

Comment:
I added the test case when the client send "get members of organization" api with search query with empty string
If there is no content (empty string) in search query parameter, it will return all members ignoring the search condition. 
Tell me if this is not the function that you expected.